src/storage-controller/src/instance.rs

```diff
@@ -65,6 +62,8 @@ pub(crate) struct Instance<T> {
     now: NowFn,
     /// Responses ready for delivery on the next call to [`Instance::recv`].
     ready_responses: VecDeque<StorageResponse<T>>,
```

Comment:
Is there a reason to keep this? Seems like we can just push responses to `instance_response_tx` instead.

Reply:
Removed